tests/integration/test_vulnerability_detector/test_scan_results/test_alert_vulnerability_removal.py

```diff
@@ -1,181 +1,294 @@
-    callback_detect_test_package_0_alert = vd.make_vuln_callback(
-        pattern=f"{test_packet_0_cve} affecting {test_packet_0_name} was eliminated", prefix='.*')
-    callback_detect_test_package_1_alert = vd.make_vuln_callback(
-        pattern=f"{test_packet_1_cve} affecting {test_packet_1_name} was eliminated", prefix='.*')
-
-    # Detect the baseline scan.
-    wazuh_log_monitor.start(timeout=vd.VULN_DETECTOR_SCAN_TIMEOUT,
-                            callback=callback_detect_baseline_scan_start,
-                            error_message='No baseline scan start has been detected in the log.')
-
-    # Check if the NVD vulnerabilities are detected.
-    vd.check_detected_vulnerabilities_number(agent=agent_id,
-                                             wazuh_log_monitor=wazuh_log_monitor,
-                                             expected_vulnerabilities_number=2,
-                                             feed_source='NVD', timeout=vd.VULN_DETECTOR_SCAN_TIMEOUT)
-
-    # Check the vulnerabilities in the inventory
-    severity, cvss2_score, cvss3_score = vd.find_rhel_cve_severity_score(cve_nvd_array=nvd_vulnerabilities['CVE_Items'],
-                                                                         cve_rhel_array=rhel_vulnerabilities,
-                                                                         cve_id=test_packet_0_cve)
-
-    vd.check_vulnerability_scan_inventory(agent=agent_id, package=test_packet_0_name, version=test_packet_version,
-                                          arch='x86_64', cve=test_packet_0_cve, condition='inserted', severity=severity,
-                                          cvss2=cvss2_score, cvss3=cvss3_score)
-
-    severity, cvss2_score, cvss3_score = vd.find_rhel_cve_severity_score(cve_nvd_array=nvd_vulnerabilities['CVE_Items'],
-                                                                         cve_rhel_array=rhel_vulnerabilities,
-                                                                         cve_id=test_packet_1_cve)
-
-    vd.check_vulnerability_scan_inventory(agent=agent_id, package=test_packet_1_name, version=test_packet_version,
-                                          arch='x86_64', cve=test_packet_1_cve, condition='inserted', severity=severity,
-                                          cvss2=cvss2_score, cvss3=cvss3_score)
-
-    # Detect baseline scan completion.
-    wazuh_log_monitor.start(timeout=vd.VULN_DETECTOR_SCAN_TIMEOUT,
-                            callback=callback_detect_scan_end,
-                            error_message='No baseline scan end has been detected in the log.')
+    # Check the vulnerabilities removal
+    check_vulnerability_scan_inventory(agent_id=agent_id, package=metadata['test_package_1_name'],
+                                       version=metadata['test_package_version_not_vulnerable'], arch='x86_64',
+                                       cve=metadata['test_package_1_cve'], condition='removed')
```

Comment:
Is this situation always true? `condition='removed'`

Reply:
In this case, yes. This will check that the element is removed from the database.

Reply:
This check has been removed